resources/js/helpers/filters.js

```diff
@@ -231,4 +231,12 @@ export function mentionify(value) {
         newValue = newValue.replace(replRegex, tpl.replace('{name}', name));
     }
     return newValue;
+}
+
+export function trim(str, charlist) {
```

Comment:
why is it a filter?

Reply:
🤷 didn't know where it fits the best. Kinda feels like a filter, but might also fit into helpers.js :D